A3A/addons/core/functions/Missions/fn_LOG_Crashsite.sqf

```diff
@@ -0,0 +1,861 @@
+        [_taskId2, "LOG", "CANCELED"] call A3A_fnc_taskSetState;
+
+        [0, 600] remoteExec ["A3A_fnc_resourcesFIA",2];
+        [1800, _sideX] remoteExec ["A3A_fnc_timingCA",2];
+        { 
+            [20*_bonus,_x] call A3A_fnc_addScorePlayer;
+            [350*_bonus,_x] call A3A_fnc_addMoneyPlayer;
+        } forEach (call SCRT_fnc_misc_getRebelPlayers);
+        [10*_bonus,theBoss] call A3A_fnc_addScorePlayer;
+        [250*_bonus,theBoss, true] call A3A_fnc_addMoneyPlayer;
+        ["Large", _sideX] remoteExec ["A3A_fnc_selectIntel", 2];
+        [(position _box), 4000, 1200, false] spawn SCRT_fnc_common_recon; ///params ["_position", "_radius", "_revealTime", ["_isInterrogation", false]]; ///revels enemy location only once, which is not good?, dunno maybe it should do it every 2-5 minutes
+        sleep 60;
+        [(position _box), 4000, 1200, false] spawn SCRT_fnc_common_recon;
+        sleep 60;
+        [(position _box), 4000, 1200, false] spawn SCRT_fnc_common_recon;
+        sleep 60;
+        [(position _box), 4000, 1200, false] spawn SCRT_fnc_common_recon;
+        deleteVehicle _box;
+	    if (hideEnemyMarkers) then {
+  		    [(selectRandom [5,7])] call A3U_fnc_revealRandomZones;
+	    };
+    };
+    case(_box distance (getMarkerPos traderMarker) < 50 || _box distance (getMarkerPos traderMarker) < 50): {
+        Info("Box has been delivered to arms traider, mission completed.");
+        [_taskId, "LOG", "CANCELED"] call A3A_fnc_taskSetState;
+        [_taskId2, "LOG", "SUCCEEDED"] call A3A_fnc_taskSetState;
+
+        [0, 1200] remoteExec ["A3A_fnc_resourcesFIA",2];
+        [3600, _sideX] remoteExec ["A3A_fnc_timingCA",2];
+        { 
+            [30*_bonus,_x] call A3A_fnc_addScorePlayer;
+            [700*_bonus,_x] call A3A_fnc_addMoneyPlayer;
+        } forEach (call SCRT_fnc_misc_getRebelPlayers);
+        [20*_bonus,theBoss] call A3A_fnc_addScorePlayer;
+        [500*_bonus,theBoss, true] call A3A_fnc_addMoneyPlayer;
+        [20] call SCRT_fnc_trader_setTraderDiscount;
+    };
+    default {
+        Error("Undefined mission outcome.");
+        [_taskId, "LOG", "CANCELED"] call A3A_fnc_taskSetState;
+        [_taskId2, "LOG", "CANCELED"] call A3A_fnc_taskSetState;
+    };
+};
+
+////////////
+//Clean up//
+////////////
+
+sleep 20;
+
+{
+    deleteVehicle _x;
+} forEach _effectsAndProps;
+
+{
+    deleteVehicle _x;
+} forEach _vehicles;
+
+[_taskId, "LOG", 1200] spawn A3A_fnc_taskDelete;
+[_taskId2, "LOG", 1200] spawn A3A_fnc_taskDelete;
+
+{[_x] spawn A3A_fnc_vehDespawner} forEach _vehicles;
+{[_x] spawn A3A_fnc_groupDespawner} forEach _groups;
+
+if (!isNil "traderMarker") then { ///checking if trader is spawned
+    if (alive _box && {_box distance (getMarkerPos respawnTeamPlayer) > 50} || _box distance (getMarkerPos traderMarker) < 50) then {
+    deleteVehicle _box;
+    };
+} else {
+    if (alive _box && {_box distance (getMarkerPos respawnTeamPlayer) > 50}) then {
+    deleteVehicle _box;

```

Comment:
both deleteVehicle statements need an indent, every potential future developer will hate you otherwise